utils/githooks/install

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -eu
+
+githook_root="${0%/*}"
+repo_root="${githook_root}/../../"
+
+pushd $repo_root > /dev/null
+for d in ${githook_root}/*.d; do
+    h=${d%.d}
+    h=${h##*/}
+    link_name=".git/hooks/$h"
+    link_target="../../utils/githooks/$h"
+    if [ -e "$link_name" ]; then
+        if [ "$(readlink $link_name)" != "$link_target" ]; then
+            echo "Link exists: $link_name"
+            echo "Either delete or move to utils/githooks/${h}.d/"
```

Comment:
There is this line in .gitignore.  If they have a custom hook that they don't want to share, then need to follow this format

utils/githooks/*.d/*-user-*